src/ContractInteract/Anchor.js

```diff
@@ -288,7 +288,7 @@ class Anchor {
    * @returns {Promise<Object>} Promise that resolves to transaction receipt.
    */
   anchorStateRoot(blockHeight, stateRoot, txOptions) {
-    if (typeof blockHeight !== 'string') {
+    if (isNaN(blockHeight)) {
```

Comment:
This should check if block height type is other than `Number` or `string`. isNaN is too general. This will pass even for an object.